<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecClass</class>
 <widget class="QMainWindow" name="ElecClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>防窃电管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="wgtHeader" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#wgtHeader{border-image: url(:/resource/header.jpg);}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>STKaiti</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>配电网防窃电智能管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgtTitle" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resource/header.png);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnSummary">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/resource/home.png</normaloff>:/resource/home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConfigure">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>系统配置</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDatabase">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>数据库配置</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="benServer">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>服务配置</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWorker">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>报警员管理</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAlarm">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>报警管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUser">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>权限管理</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuit">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1799</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 20, 50);</string>
         </property>
         <property name="text">
          <string>弹窗测试</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pgSummary">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SummaryWidget" name="wgtSummary" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgConfigure">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="CenterWidget" name="wgtConfigure" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CenterWidget</class>
   <extends>QWidget</extends>
   <header location="global">CenterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SummaryWidget</class>
   <extends>QWidget</extends>
   <header location="global">SummaryWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
